### ISSUE TITLE
LibreTube blocked by captcha(placed by cdn) when it trys to connect to instance

### ISSUE THREAD

**Issue by happyeggchen** [libre-tube/LibreTube#1981]
### Steps to reproduce

1. You have a flagged ip address by cdn
2. Visit instance on browser, captcha poped up
3. Connect to instance using libretube, libretube reports that the instance is not available

### Expected behavior

Be able to popup a window and let user confirm that they're human

### Actual behavior

reports "instance is not available"

### LibreTube version

0.7.0

### Android version

A12.1 Crdroid

### Other details

Maybe we can make a popup to let user confirm that they're human
A third party nhentai android client did this

https://github.com/Dar9586/NClientV2/pull/428

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
There are no plans to add that inside the app, there are enough instances that don't use CDN!

**Comment by happyeggchen** [comment]
Yes....But i'm using my own instance so..
Thanks anyway i'll remove my instance from cdn and connect to it directly
